src/frontend/templates/footer.html

```diff
@@ -32,6 +32,9 @@
                     <b>Cluster: </b>{{index .deploymentDetails "CLUSTERNAME" }}<br/>
                     <b>Zone: </b>{{index .deploymentDetails "ZONE" }}<br/>
                     <b>Pod: </b>{{index .deploymentDetails "HOSTNAME" }}
+                    {{ else }}
+                    Deployment details are either still loading or failed to load.
```

Comment:
Not too opiniated on this but I think we should silently fail and not show anything rather than show an error message. Or just show "Deployment details are unavailable" or similar.
The reasoning behind this is that more often than not, the deployment details are simply not available, but the current diff makes it seem like the user _should_ be reloading, or that there is something fundamentally wrong with their cluster's well being.
Thoughts?

Reply:
@ckim328 suggested a similar "fail silently" user experience (i.e., hiding labels that have no values).
So I've gone ahead and implemented that. :)

I have removed the following verbiage: 
>  or failed to load

because this was not true. If an error occurred, we would've just seen labels with no values.

I have preserved the following verbiage:
> Deployment details are still loading.
> Try refreshing this page.

because good error message are actionable.
And there is a tutorial that asks developers to look at the deployment details (see #628).

Thanks, you two.